build-system/tasks/report-test-status.js

```diff
@@ -34,37 +37,42 @@ const TEST_TYPE_SUBTYPES = new Map([
   // TODO(danielrozenberg): add 'saucelabs' to integration tests when supported.
   ['integration', ['local', 'single-pass']],
   ['unit', ['local', 'local-changes', 'saucelabs']],
-  // TODO(danielrozenberg): add 'e2e' tests.
+  ['e2e', ['local']],
 ]);
 const TEST_TYPE_BUILD_TARGETS = new Map([
   ['integration', ['RUNTIME', 'FLAG_CONFIG', 'INTEGRATION_TEST']],
   ['unit', ['RUNTIME', 'UNIT_TEST']],
```

Comment:
`['e2e', ['RUNTIME', 'FLAG_CONFIG', 'E2E_TEST']],`

Reply:
The sins of rebasing and removing commits >__<

Done.